.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -1748,5 +1772,14 @@ s3_Scenario_GettingStarted:
             - description:
               snippet_tags:
                 - ruby.example_code.s3.Scenario_GettingStarted
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rust_dev_preview/s3
+          excerpts:
+            - description:
+              snippet_tags:
+                - rust.example_code.s3.scenario_getting_started.bin
+                - rust.example_code.s3.scenario_getting_started.lib

```

Comment:
Suggestion: you can list the excerpts separately with descriptions in between. In this case, I think that might look better in the rendered output. 

Something like this, but with better descriptions:
```
          excerpts:
            - description: Code for running the scenario.
              snippet_tags:
                - rust.example_code.s3.scenario_getting_started.bin
            - description: Single-action calls used by the scenario.
              snippet_tags:
                - rust.example_code.s3.scenario_getting_started.lib
```

Reply:
Excellent, I was wondering the best way to do that. Good suggestion.